### DISCUSSION TITLE
Power mode on battery

### DISCUSSION THREAD

**Discussion by Autobiography59** [seerge/g-helper#1597] [Q&A]
Hello everyone, how to always stay on manual mode? Even on battery mode

**Comment by Konnos998** [comment]
Just make a new profile (for example rename it Manual mode) choose this profile while on battery, and also when you plug your laptop in on AC/socket. That's it.
On battery it won't be as good tho, because it won't work on it's full potential wattage (like it would on AC). You'll be able to hear the fans working harder, but the performance won't be as good as on AC, because you won't have maximum wattage for the CPU and GPU.
